stormlog/phases/runtime.py

```
@@ -0,0 +1,314 @@
+
+__all__ = [
+    "PHASE_ENTER_EVENT",
+    "PHASE_EXIT_EVENT",
+    "PHASE_SCOPE_METADATA_KEY",
+    "PHASE_SCOPE_ATTRIBUTES_KEY",
+    "PhaseBoundary",
+    "PhaseHandle",
+    "PhaseProtocolError",
+    "PhaseRecorder",
+    "PhaseToken",
+]
```

Ruff (0.15.10)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)